### PR TITLE
minor readability update

### PR DESCRIPTION
**Pull request by Farbum**
minor readability update

---

> [!NOTE]
> Fix unmask logic to set non-missing masks to ONLINE_ENCODER while preserving MISSING values.
> 
> - **Masking**:
>   - Update `MaskedOlmoEarthSample.unmask()` to correctly set all non-missing mask entries to `MaskValue.ONLINE_ENCODER` while preserving `MaskValue.MISSING`, using `torch.where` in `olmoearth_pretrain/train/masking.py`.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit e07c96d8c43911527dc208d95d7f9de75e1aab8e. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>

### CODE REVIEW

**Comment by gabrieltseng** [comment]
okay so it was a bug... 

I added some tests to catch it here: https://github.com/allenai/olmoearth_pretrain/pull/457
